ui/app/styles/components/ui-wizard.scss

```diff
@@ -1,3 +1,6 @@
+$progress-bar-height: 6px;
+$progress-check-size: 16px;
```

Comment:
I feel a little funny adding stuff like this to `bulma_variables`, but I don't like putting them here either. What do you think?

Reply:
Goooood q. I think here is fine? Do you see us using them elsewhere?

Reply:
Nope, so I should namespace it better for sure. We do have some variables in `bulma_variables` that aren't really for Bulma, so maybe renaming or separating those is the real question.

Reply:
Moved them to `bulma_variables` for now since we already have some in there and don't want to hold this up. Added better namespaced variables too

Reply:
I think having them here isn't a bad thing since the progress bar only lives here - but maybe wizard-progress as a prefix? I know we used to have a progress bar on unseal, but that's now a radial progress ... just thinking a collision would be hard to track down 😂